### DISCUSSION TITLE
2FA / MFA

### DISCUSSION THREAD

**Discussion by Br0kenSilos** [TriliumNext/Notes#288] [Ideas]
I don't know how much other people would want or need this feature. But personally I would love to see some form of 2fa / mfa added to the web UI. This would provide added security for users who expose their instance to the web through one means or another. Make it optional so it can be turned on / off. 2FA options could be TOTP (via the various apps available - 2fas, bitwarden, Google authenticator, etc..) with QR codes, and/or something like DUO, or even passkeys.

**Comment by meichthys** [comment]
https://github.com/TriliumNext/Notes/pull/163

**Reply by Br0kenSilos** [reply]
I don't know how I missed that one.
